cmd/podman/rm.go

```diff
@@ -67,7 +67,7 @@ func rmCmd(c *cliconfig.RmValues) error {
 	}
 
 	if len(failures) > 0 {
-		exitCode = 125
+		exitCode = 2
```

Comment:
shouldn't we check here that the reason is the container running/paused and not another kind of failure?

Reply:
Yes the only time we should return a non 125 exit code is if the container or image does not exists.